py-polars/tests/unit/io/files/single_column.csv

```diff
@@ -0,0 +1,4 @@
+a
```

Comment:
Can you make this file in the test?

You can put that string in a file buffer and read that in the test.

Reply:
Yes, I have made the commit for the change.
I also noticed another thing that needs testing, when the file ends without LF. So, I've added that to the update.